src/components/agenda/AgendaSideBar.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useRef } from 'react';
+
+function AgendaSideBar({updateAgenda}) {
+  const titleRef = useRef('')
+  const descriptionRef = useRef('')
+  const [time, setTime] = useState('')
+
+  const onSubmit = (e) => {
+    e.preventDefault()
+    const newAgendaItem = formatAgendaItem()
+    updateAgenda(newAgendaItem)
+    titleRef.current.value = ''
+    descriptionRef.current.value = ''
+    setTime('')
+  }
+
+  const convertTime = (time) => {
+    const splitTime = time.split(':')
+
+    const hours = Number(splitTime[0])
+    const minutes = Number(splitTime[1])
+
+    let timeValue;
+
+    if (hours > 0 && hours <= 12) {
+      timeValue = "" + hours;
+    } else if (hours > 12) {
+      timeValue = "" + (hours - 12);
+    } else if (hours === 0) {
+      timeValue = "12";
+    }
+
+    timeValue += (minutes < 10) ? ":0" + minutes : ":" + minutes
+    timeValue += (hours >= 12) ? "PM" : "AM"
```

Comment:
This looks really great. I like how you've set this up. If we have time down the line might be worth splitting some of these into a utils file.